src/Compilers/CSharp/Test/Symbol/Symbols/ReplaceOriginalTests.cs

```diff
@@ -1,29 +1,26 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using System.Linq;
-using System.Threading;
 using Microsoft.CodeAnalysis.CodeGen;
 using Microsoft.CodeAnalysis.CSharp.Symbols;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
 using Microsoft.CodeAnalysis.Semantics;
+using Microsoft.CodeAnalysis.Test.Utilities;
 using Roslyn.Test.Utilities;
+using System.Linq;
+using System.Threading;
 using Xunit;
 
 namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Symbols
 {
     public class ReplaceOriginalTests : CSharpTestBase
     {
-        private static CSharpCompilation CreateCompilationWithMscorlib(string text)
-        {
-            return CreateCompilationWithMscorlib(text, parseOptions: TestOptions.Regular.WithReplaceFeature());
-        }
-
-        private static CSharpCompilation CreateCompilationWithMscorlib(string text, CSharpCompilationOptions options)
+        private static CSharpCompilation CreateCompilationWithMscorlib(string text, CSharpCompilationOptions options = null)
         {
             return CreateCompilationWithMscorlib(text, options: options, parseOptions: TestOptions.Regular.WithReplaceFeature());
         }
 
+        [CompilerTrait(CompilerFeature.SourceGenerators)]
```

Comment:
I thought we could simply mark the type and avoid the need to mark individual tests within it.